var/spack/repos/builtin/packages/g2c/package.py

```diff
@@ -16,23 +16,22 @@ class G2c(CMakePackage):
 
     maintainers = ['kgerheiser', 'Hang-Lei-NOAA', 'edwardhartnett']
 
+    version('1.6.4', sha256='5129a772572a358296b05fbe846bd390c6a501254588c6a223623649aefacb9d')
+    version('1.6.2', sha256='b5384b48e108293d7f764cdad458ac8ce436f26be330b02c69c2a75bb7eb9a2c')
+
```

Comment:
Just fixing the ordering here with versions on top

Reply:
Thanks!